.github/workflows/python-package.yml

```diff
@@ -36,16 +36,11 @@ jobs:
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is [supposed](https://github.com/marketplace/actions/coveralls-github-action) to replace the repo-token...but I don't think it is....